pkg/controller/gittrack/gittrack_controller.go

```diff
@@ -272,7 +272,7 @@ func (r *ReconcileGitTrack) listObjectsByName(owner *farosv1alpha1.GitTrack) (ma
 	return result, nil
 }
 
-// result represents the result of creating or updating a GitTrackObject
+// result represents the result of creating or updating a GitTrack
```

Comment:
This one is actually correct, this controller creates and updates `GitTrackObjects` and `ClusterGitTrackObjects`